iron-swipeable-pages.html

```diff
@@ -253,6 +261,10 @@
 
         // prepare the page for animation and add it to the list of pages to be animated
         _initPage: function(page, transition, left) {
+          if (!page) {
+            return;
+          }
+
```

Comment:
can you create another PR for this one, so that I can merge the maximum duration first and do some test with the initPage issue?